### DISCUSSION TITLE
LEDs continue to operate after system shutdown

### DISCUSSION THREAD

**Discussion by Naglietroll** [awawa-dev/HyperHDR#940] [Q&A]
Good time!!!

I would like to thank the author for his hard work and what he is doing.
I decided to upgrade to HyperHDR, after upgrading the current build everything works. But there are a few problems. The most basic is that after disconnecting the set-TV box, from which the signal comes, the LEDs continue to burn a weak white color. The assembly is as follows: TV box Nvidia, from it goes HDMI to EZCOO splitter, from there to Raspberry 4 and Samsung TV, from TV to Samsung soundbar (I don't know if it plays a role what I use). It's running an Adafruit Feather RP2040 SCORPIO LED strip. Perhaps I can provide some more information to solve the problem?
The second problem is that HyperHDR doesn't start automatically after a system reboot. I set the username “pi” and password “raspberry” when reinstalling the Raspberry OS, but it didn't help.
I have done a search for similar discussions, but even in those there is no solution to my problems. Any help would be greatly appreciated.

**Comment by awawa-dev** [comment]
Hi

> I decided to upgrade to HyperHDR, after upgrading the current build everything works. But there are a few problems. The most basic is that after disconnecting the set-TV box, from which the signal comes, the LEDs continue to burn a weak white color. The assembly is as follows: TV box Nvidia, from it goes HDMI to EZCOO splitter, from there to Raspberry 4 and Samsung TV, from TV to Samsung soundbar (I don't know if it plays a role what I use). It's running an Adafruit Feather RP2040 SCORPIO LED strip. Perhaps I can provide some more information to solve the problem?

Set Backlight threshold to 0 in the processing image tab

> The second problem is that HyperHDR doesn't start automatically after a system reboot. I set the username “pi” and password “raspberry” when reinstalling the Raspberry OS, but it didn't help.

Dont know what can be a problem... you can try to to remove HyperHDR and install it again (after the restart) using APT repository https://awawa-dev.github.io/ Before installing it make sure there is no hyperhdr process (ps command)

**Reply by Naglietroll** [reply]
It's magic. I've been going over the CEC settings all evening trying to get the LEDs to turn off completely and no result. The developer comes and says to change 1 to 0 and everything works. Thank you very much.

I reinstalled HyperHDR from the repository at the link, it didn't make a difference, HyperHDR still doesn't restart automatically after OS reboot. But I'm so happy with the first problem and the way it all works and looks, that sometimes starting the service manually doesn't seem to be a problem anymore. Thanks again.

Translated with DeepL.com (free version)

**Reply by awawa-dev** [reply]
> It's magic. I've been going over the CEC settings all evening trying to get the LEDs to turn off completely and no result. The developer comes and says to change 1 to 0 and everything works. Thank you very much.

Minimal brightness is enabled on default for some reason: to minimize flickering. You can leave it disabled or configure no-signal detection in the grabber section instead.

> I reinstalled HyperHDR from the repository at the link, it didn't make a difference, HyperHDR still doesn't restart automatically after OS reboot. But I'm so happy with the first problem and the way it all works and looks, that sometimes starting the service manually doesn't seem to be a problem anymore. Thanks again.

Try this if your user is pi:

```
sudo systemctl disable --now hyperhdr@pi
sudo systemctl disable --now hyperhdr@root
sudo systemctl enable --now hyperhdr@pi
sudo reboot
```

if still doesnt work please provide `sudo systemctl status hyperhdr@pi`  and journalctl logs

**Comment by Naglietroll** [comment]
Thank you.

> Minimal brightness is enabled on default for some reason: to minimize flickering. You can leave it disabled or configure no-signal detection in the grabber section instead.

Do I understand correctly that this is the Automatic signal detection setting on the Video capturing tab? Should I disconnect the HDMI cable from the grabber and press calibrate? Do I need to change any of the settings that appear below? I just did the calibration without HDMI cable, returned one instead of zero in Backlight threshold and everything works fine.

> Try this if your user is pi:

After entering the above commands, HyperHDR started to run automatically. But when entering the UI, the error [SERVICE_mDNS] Failed to open any client sockets pops up, but it has no effect on HyperHDR operation.

**Comment by awawa-dev** [comment]
> Do I understand correctly that this is the Automatic signal detection setting on the Video capturing tab? Should I disconnect the HDMI cable from the grabber and press calibrate? Do I need to change any of the settings that appear below? I just did the calibration without HDMI cable, returned one instead of zero in Backlight threshold and everything works fine.

You should calibrate it for the conditions you described earlier:
`The most basic is that after disconnecting the set-TV box, from which the signal comes, the LEDs continue to burn a weak white color.`

**Reply by Naglietroll** [reply]
Yes, that's what I did and it works great. Thanks again.
